tests/config/test_data.config

```diff
@@ -555,6 +556,60 @@ params {
                 openms_idxml                    = "${params.test_data_base}/data/proteomics/openms_idxml/BSA_QC_file.idXML"
             }
         }
+        'mus_musculus' {
```

Comment:
You still have a lot of changes in this file. I'm assuming you don't actually intend all of them?

Reply:
I really can't figure out what went wrong here :( ... I tried a bit of git magic but it didn't work, i copied/pasted the file from the master branch on the nf-core repo and linted it but some spaces are still lingering around, is that ok?

Reply:
Looks better now, seems like you're just removing a few rogue spaces, which is okay.